### PR TITLE
[Keyboard] Add chord/zero

### PR DESCRIPTION
**Pull request by sol**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/chord/zero/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    keymaps[0][0][0],           keymaps[0][4][0],
+    keymaps[0][1][0],           keymaps[0][5][0],
+        keymaps[0][3][3],   keymaps[0][7][3]
+, COMBO_END };
+
+combo_t key_combos[] = {
+    COMBO(bootloader_combo, QK_BOOTLOADER),
+};
+
+uint16_t COMBO_LEN = sizeof(key_combos) / sizeof(key_combos[0]);
```
Comment (drashna) [inline keyboards/chord/zero/keymaps/default/keymap.c:25]: Is this actually needed? 
```suggestion
```

Comment (sol) [reply keyboards/chord/zero/keymaps/default/keymap.c:25]: If I remove it I get a link error. So I guess yes, it's needed.

I took this approach from the documentation here: https://github.com/qmk/qmk_firmware/blob/master/docs/feature_combo.md#variable-length-combos

In this case it's just a fancy way of setting `COMBO_LEN` to one. I still think this approach is quite neat.

Comment (drashna) [reply keyboards/chord/zero/keymaps/default/keymap.c:25]: ah, okay.  That's something that should probably be refactored. 😅

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by sol** [comment]
I just noticed that the image in `readme.md` pointed to an outdated (overly large in size) version.  I updated that and force pushed.

From my side it's good to go now.

File: keyboards/chord/zero/readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# CHORD ZERO Stenographic Keyboard
+
+![CHORD ZERO](https://user-images.githubusercontent.com/461132/187019809-5aaccf27-c189-4561-801b-86f32a59df2b.jpg)
```
Comment (drashna) [inline keyboards/chord/zero/readme.md:3]: Please rehost on Imgur or the like.

Comment (sol) [inline keyboards/chord/zero/readme.md:3]:
```suggestion
![CHORD ZERO](https://i.imgur.com/aS2MPQ2h.jpg)
```

Comment (sol) [reply keyboards/chord/zero/readme.md]: Done ✔️

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**